hubblestack/daemon.py

```diff
@@ -683,6 +683,9 @@ def _setup_dirs():
     """
     Setup module/grain/returner dirs
     """
+    hubble_mods_dirs = __opts__.get('hubble_mods_dirs', [])
```

Comment:
We don't need to include extmods/audit and extmods/fdg folders now. Need to remove them.

Reply:
will remove fdg folder when its implemented